pages/common/fastboot.md

```diff
@@ -0,0 +1,27 @@
+# fastboot
+
+> Communicate to connected Android devices when in bootloader mode (the one place adb doesn't work).
+
+- Unlock bootloader:
+
+`fastboot oem unlock`
+
+- Re-lock the bootloader:
+
+`fastboot oem lock`
+
+- Reboot device from fastboot mode to fastboot mode again:
+
+`fastboot reboot bootloader`
+
+- Flash ASDF.zip:
+
+`fastboot flash {{ASDF.zip}}`
+
+- Flash a custom recovery image ASDF.img:
+
+`fastboot flash recovery {{ASDF.img}}`
+
+- Display connected devices:
```

Comment:
Perhaps `Display the connected`.... would read better?

Reply:
I think it should be the same as it already makes sense. Doesn't it?

Reply:
It is understandable, yes - but it's more grammatically correct to say `Display the connected devices:`.